code/game/objects/items/carp_lasso.dm

```diff
@@ -0,0 +1,44 @@
+/obj/item/carp_lasso
+	name = "carp lasso"
+	desc = "Comes standard with every space-cowboy."
+	icon = 'icons/obj/carp_lasso.dmi'
+	///Ref to timer
+	var/timer
+	///Ref to lasso'd carp
+	var/mob/living/simple_animal/hostile/carp/carp_target
```

Comment:
Make sure this doesn't cause harddels

Reply:
No worries, smokey
![image](https://user-images.githubusercontent.com/40559528/177014493-4bb7829a-c948-400a-aa8d-8fdd0c4b5b3a.png)